devel/irrlichtmt/Portfile

```diff
@@ -19,3 +20,29 @@ checksums           rmd160  4da410c24f67e8c0ebd9880308b8d796d2f86676 \
 depends_lib         path:include/turbojpeg.h:libjpeg-turbo \
                     port:libpng \
                     port:zlib
+
+pre-build {
```

Comment:
[`mesa`](https://ports.macports.org/port/mesa/) already includes both headers. It maybe better to copy a fragment from Minetest:

`depends_build    port:mesa`

Reply:
THX

Please be aware there was an argument that I always had to heed the work of my (unknown) predecessor as maintainer and that to always include "all the paths" to me as the new maintainer. during rev1.
AFAIK see @reneeotten in #17080 if I reall this recent discussion correctly.

Furthermore, this "mess" basically started with my innocent idea of dropping "mesa" and GLES completely by kind advice from @l2dy and discussion at:
- https://trac.macports.org/ticket/66599

@aeiouaeiouaeiouaeiouaeiouaeiou  To better disambiguate: By a.m. "both headers" you mean the top two of three headers i.e. "libjpeg-turbo" and "libpng" I presume?

Reply:
Might drop both headers i.e. "libjpeg-turbo" and "libpng" in due time.

Reply:
Anyway, why should I drop both headers i.e. 'libjpeg-turbo' and 'libpng' from "irrlichtmt" port?

@l2dy is the "irrlichtmt" maintainer and I am the just the new maintainer to "minetest" providing some fixes after discussion with same "irrlichtmt" maintainer.

Please note I am **quite reluctant to gain too much influence** on other people's ports and I will always appreciate their efforts as maintainers of their port and distribution to FOSS.

Reply:
>@aeiouaeiouaeiouaeiouaeiouaeiou To better disambiguate: By a.m. "both headers" you mean the top two of three headers i.e. "libjpeg-turbo" and "libpng" I presume?

I meant the header files - glext.h and khrplatform.h, which are attached to the commit.

Reply:
> > @aeiouaeiouaeiouaeiouaeiouaeiou To better disambiguate: By a.m. "both headers" you mean the top two of three headers i.e. "libjpeg-turbo" and "libpng" I presume?
> 
> I meant the header files - glext.h and khrplatform.h, which are attached to the commit.

THX

Please have a look on the conditionals in a.m. `pre-built` script ans you could be aware:
- 'glext.h' is only applied to l**egacy-os** platforms Snow Leopard and maybe Leopard (afaik the latter fails due to other reasons unfortunately)
- 'khrplatform.h' applies only to **arm64** platforms for obvious reasons.

Reply:
@aeiouaeiouaeiouaeiouaeiouaeiou THX and you have a good heart.